modules/policy/engine/src/main/java/org/eclipse/sw360/antenna/policy/engine/PolicyException.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+/**
+ * Policy {@link Rule} execution in a {@link RuleExecutor} ends with this exception if a policy violation is found.
+ */
+public class PolicyException extends Exception {
+    public PolicyException() {
```

Comment:
All functions throwing `PolicyException` have `void` return type, and thus could encode the result in the return value instead.